src/Perlang.Parser/Scanner.cs

```diff
@@ -34,13 +33,16 @@ public class Scanner
                 { "this", THIS },
                 { "true", TRUE },
                 { "var", VAR },
-                { "while", WHILE }
+                { "while", WHILE },
+
+                // Reserved keywords
+                { "class", RESERVED_WORD }, // Pending #66 to be resolved.
```

Comment:
This PR turned out to be way harder than I first thought (are we surprised that this is happening _again_? :wink:). I had initially intended to add all the reserved words here, but this causes problems since some of them (`int` and `string` in particular) are actually usable as type names for variables and return types. By making them be `RESERVED_WORD` here, the parser will fail to build a valid syntax tree for these use cases.

The solution is to add only the other reserved words here, and handle the type names specially, namely...